src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -92,6 +92,26 @@ private static CSharpCommandLineArguments FullParse(string commandLine, string b
         }
 
         [Fact]
+        public void RunWithShiftJisFile()
```

Comment:
Can we have another version of this test which doesn't explicitly pass the code page and only runs when ACP is 932?  That will give us as close to a developer experience as possible when we do our weekly localization runs.